yapapi/golem.py

```diff
@@ -265,7 +265,7 @@ async def worker(context: WorkContext, tasks: AsyncIterable[Task]):
                 image_hash="d646d7b93083d817846c2ae5c62c72ca0507782385a2e29291a3d376",
```

Comment:
I can't reach that far up, but we can update the command results usage here as well!